ros_cross_compile/platform.py

```diff
@@ -84,9 +84,11 @@ def __init__(
         else:
             raise ValueError('Unknown ROS distribution "{}" specified'.format(ros_distro))
 
-        if self.os_name not in ROSDISTRO_OS_MAP[self.ros_distro]:
+        os_map = ROSDISTRO_OS_MAP[self.ros_distro]
+        if self.os_name not in os_map:
             raise ValueError(
-                'OS "{}" not supported for ROS distro "{}"'.format(os_name, ros_distro))
+                'OS "{}" not supported for ROS distro "{}". Supported values: {}'.format(
```

Comment:
Although it's likely that a user can put two and two together to see that the OS names are case sensitive, it might not be a bad idea to include a note that they are case sensitive. If not here, then in the README for the project.

Reply:
good call :+1: